packages/skills/weather_station/dummy_weather_station_data.py

```diff
@@ -38,44 +60,47 @@
 
 
 class Forecast():
+    """Represents a whether forecast."""
+
+    def add_data(self, tagged_data: Dict[str, Union[int, datetime.datetime]]) -> None:
```

Comment:
We could think of using some ORM for Python like [SQLAlchemy](https://www.sqlalchemy.org/)